index.js

```diff
@@ -11,46 +11,57 @@ let total = 0;
 let counter = 0;
 const template = $('#hidden-template').html();
 
-for (var version of java_version) {
+function request(url) {
+  const xmlHttp = new XMLHttpRequest();
+  xmlHttp.open("GET", url, false);
+  xmlHttp.send(null);
+  return JSON.parse(xmlHttp.response);
+}
+
+// Dockerhub pulls
+let result = request("https://cors-anywhere.herokuapp.com/https://hub.docker.com/v2/repositories/adoptopenjdk?page_size=100")
+for (let repo of result.results) {
+  total += repo.pull_count
+}
+
+// Github release downloads
+for (let version of java_version) {
   let item = $(template).clone();
   $(item).find('.count-text').html(version);
   $(item).find('#link').attr("href", "./release.html?version=" + version);
   $(item).css("display", "none");
   $('#versions').append(item);
-  var settings = {
-    "url": `https://api.adoptopenjdk.net/v2/info/releases/${version}`,
-    "method": "GET"
+
+  let result = request(`https://api.adoptopenjdk.net/v2/info/releases/${version}`)
```

Comment:
What type of result? More accurate var name might help here.

Reply:
renamed